db/migrate/20230117220451_add_foreign_key_to_comment.rb

```diff
@@ -0,0 +1,6 @@
+class AddForeignKeyToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :author_id, :integer
+    add_column :comments, :post_id, :integer
+  end
```

Comment:
- [ ] Kindly add referential integrity to the comments table unlike the columns which you have added.
example
![image](https://user-images.githubusercontent.com/81932478/213059720-e6f20a98-8d68-4840-aea1-d4cf551791bf.png)